### PR TITLE
LPS-166758 Improvements in keyboard movement

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
Hey @sandrodw3,

What is "f"?

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
@sandrodw3 there is a SF error

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/KeyboardMovementText.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import React from 'react';
+
+import {useMovementText} from '../contexts/KeyboardMovementContext';
+
+export default function KeyboardMovementText() {
+	const text = useMovementText();
+
+	return text ? (
+		<span aria-atomic="true" aria-live="assertive" className="sr-only">
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/KeyboardMovementText.js:23]: Here you can use `role="alert"`, is equivalent to `aria-live="assertive"` and `aria-atomic="true"`
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/KeyboardMovementText.js:23]: Cool! I didn't know it 😄 I'm applying the change

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**